docs/contributing/STYLE-GUIDE.md

```diff
@@ -1,4 +1,4 @@
-# Fossa CLI style guide
+# FOSSA CLI style guide
```

Comment:
Should it be `fossa-cli` even if it's in a title?

Reply:
It _could_, but I think this reads better.  Maybe "Style Guide for the FOSSA CLI" so that it's not a proper noun.